samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Bookmarks.razor

```diff
@@ -12,7 +12,7 @@
     Sample demonstrates the use of the Bookmarks widget and Bookmark actions.
     This follows the ESRI sample, but the Timeline Widget is included with GeoBlazor pro.
 </p>
-<MapView @ref="_mapView" Class="map-view" OnLayerViewCreate="OnLayerViewCreate">
+<MapView Class="map-view" OnLayerViewCreate="OnLayerViewCreate">
```

Comment:
I see a number of the samples pages will no longer using the MapView, is this a refactor that we will want to do across most, if not all layer types going forward? For Consistency